public/static/themes/simple.css

```
@@ -1,4 +1,5 @@
 /* @import url('./catppuccin-mocha.css'); */
+@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600;700;800&display=swap');
```

```suggestion
@font-face {
  font-family: 'Rubik';
  src: url('https://fonts.googleapis.com/css2?family=Rubik:wght@400;500;600;700;800&display=swap');
  fallback: sans-serif;
}
```

I think I would suggest using the @font-face property instead of using the @import because it tends to provide many options which can be used to improve the speed of loading the website.

You can find more information on @font-face property on the docs (link attached below) which I would suggest going through :slightly_smiling_face: :

https://developer.mozilla.org/en-US/docs/Learn/Performance/CSS#font_loading